docs/book/src/tasks/certs/experimental-ca-rotation.md

```diff
@@ -0,0 +1,112 @@
+## Experimental CA rotation procedure
+
+<aside class="note warning">
+
+<h1>Cluster-api CA rotation support</h1>
+
+CA rotation is not supported (nor automated) by cluster-api, please use this procedure at your own risk!
+Rotating CA is a tough Kubernetes topic on its own, it's even harder when everything is wrapped around cluster-api automation and immutability concepts.
+
+An issue is currently tracking discussions around this feature request : [Automatic CA rotation in CAPI #7721](https://github.com/kubernetes-sigs/cluster-api/issues/7721). Reading the issue is strongly advised to really be sure of what you are going to do.
+
+Cluster-api ultimate goal is to support such feature. But at the moment, the best (and cleaner) strategy is still to destroy and recreate your managed cluster when the CA is expired.
```

Comment:
```suggestion
The recommended strategy in CAPI is to destroy and recreate clusters ahead of the expirty date to avoid CA expiry.
```
```suggestion
Cluster-api ultimate goal is to support such feature. But at the moment, the best (and cleaner) strategy is still to destroy and recreate your managed cluster when the CA is expired.
```